src/utils/objectPath.js

```diff
@@ -70,7 +70,7 @@ export function deletePath(obj, path, { deleteEmptyObjects = false } = {}) {
     const shouldDeleteKey =
       deleteEmptyObjects === true ||
       (Array.isArray(deleteEmptyObjects) &&
```

Comment:
Should it be `Array.isArray(deleteEmptyObjects.except)`?

Reply:
yes, had deleted testing something and forgot to add back